addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2501,7 +2501,7 @@ const Wysiwyg = Widget.extend({
                 });
                 dialog.open({shouldFocusButtons:true});
 
-                this.resetEditor(dbRecord.body);
+                await this.resetEditor(dbRecord.body);
```

Comment:
Are you sure this is the _only_ call to `resetEditor` ? I would have expected it to be called more than once and thus see more than one added `await` in the diff.

Reply:
It is called twice. The second call is in `updateWidget`. That call should not make owl wait for the peer to peer promise to be loaded.